src/components/gallery/HdGalleryCarousel.vue

```diff
@@ -150,7 +152,10 @@ export default {
     },
     updateCurrentIndex(itemIndex) {
       this.currentIndex = itemIndex;
-      if (window.matchMedia('min-width: $break-tablet')) {
+
+      const mediaQuery = `(max-width: ${sassVariables.breakTablet})`;
```

Comment:
Only thing I'd suggest here, for the sake of documentation, is naming this variable more specifically, and adding a comment next to the if condition which explains the need for this "magical" condition.  Something like:

```
const tabletMediaQuery = `(max-width: ${sassVariables.breakTablet})`;

// Should not emit on desktop, as Flickity behaves differently there. 
// On desktop, the index represents the page number, not the active image. See [link to docs here?]. 
// Prevents this bug [link to bug ticket here?] 
if (mediaMatches(tabletMediaQuery)) { ... }
```

Thoughts?

Reply:
thanks for the review and the suggestions, I totally agree 👌